src/commands/app/dev/index.js

```diff
@@ -16,27 +16,44 @@ const fs = require('fs-extra')
 const https = require('https')
 const getPort = require('get-port')
 const open = require('open')
+const dedent = require('dedent')
 
 const { Flags, ux } = require('@oclif/core')
 const coreConfig = require('@adobe/aio-lib-core-config')
 
-const BaseCommand = require('../../BaseCommand')
-const runDev = require('../../lib/run-dev')
-const SERVER_DEFAULT_PORT = 9080
-// const { runInProcess } = require('../../lib/app-helper')
+const BaseCommand = require('../../../BaseCommand')
+const runDev = require('../../../lib/run-dev')
 
+const SERVER_DEFAULT_PORT = 9080
 const DEV_KEYS_DIR = 'dist/dev-keys/'
 const PRIVATE_KEY_PATH = 'dist/dev-keys/private.key'
 const PUB_CERT_PATH = 'dist/dev-keys/cert-pub.crt'
 const CONFIG_KEY = 'aio-dev.dev-keys'
 
 class Dev extends BaseCommand {
   async run () {
-    // cli input
     const { flags } = await this.parse(Dev)
 
+    if (!flags['any-terminal'] && process.env.TERM_PROGRAM !== 'vscode') {
+      const message =
+        dedent`
+        This command should only be run in a Visual Studio Code debug context.
+        Run 'aio app dev init' to initialize your App Builder project, and use the VS Code Debugger.`
+      this.error(message)
+    }
```

Comment:
per Moritz's comment: remove this restriction.

Reply:
fixed in https://github.com/adobe/aio-cli-plugin-app-dev/pull/14/commits/71a982cd9b58d14d6b58607e52c6d86c5d1044f8